lessons/beginners/tuple/index.md

```diff
@@ -0,0 +1,179 @@
+## <var>N</var>-tice
+
+Když už známe seznam, podívejme se na jeho sestřičku: takzvanou
+<var>n</var>-tici (angl. *tuple*).
+
+<var>N</var>-tice, podobně jako seznam,
+může obsahovat <var>n</var> prvků. 
+<var>N</var>-tice se dvěma prvky je *dvojice*
+neboli *pár* (angl. *pair*); se třemi
+prvky *trojice* (angl. *3-tuple*),
+se čtyřmi *čtveřice* (angl. *4-tuple*), atd.
+
+!! note ""
+    Existují i <var>n</var>-tice s jedním prvkem (hmm... „jednice”?)
+    a s nula prvky (prázdné <var>n</var>-tice, angl. *empty tuple*),
+    ale těmi se ze začátku nebudeme zabývat.
+
+<var>N</var>-tice se tvoří jako seznamy, jen kolem sebe nemají hranaté závorky.
+Stačí čárky mezi prvky.
+
+Chovají skoro stejně jako seznamy, jen nejdou měnit.
+Nemají tedy metody jako `append`
+a `pop`, a nedá se jim přiřazovat do prvků.
+Dají se ale použít v cyklu `for`,
+a dájí se z nich brát jednotlivé prvky.
+
+```python
+osoby = 'máma', 'teta', 'babička'
+for osoba in osoby:
+    print(osoba)
+print('První je {}'.format(osoby[0]))
+```
+
+!!! note ""
+Vypadá to povědomě? Aby ne!
+<var>N</var>-tice jsme už použil{{gnd('i', 'y', both='i')}} dříve:
+`for jmeno in 'Hynek', 'Vilém', 'Jarmila':`
+ve skutečnosti používá <var>n</var>-tici!
+
+Když chceš <var>n</var>-tici předat funkci,
```

Comment:
nejde poznat co předáváme čemu (ach ta čeština). Co třeba "do funkce"? Vím, že to není ideální, ale aspoň je to jasnější.